ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_creation.py

```diff
@@ -548,3 +549,35 @@ def test_jax_numpy_full(
         fill_value=fill_value,
         dtype=dtype,
     )
+
+
+def test_jax_numpy_double(
+    *,
+    dtype_and_x,
+    as_variable,
+    num_positional_args,
+    native_array,
+    on_device,
+    fn_tree,
+    frontend,
+):
+    # Check if the input datatype is double
+    if dtype_and_x[0] != "float64":
+        return
+    # Generate test inputs
+    input_dtype, x = dtype_and_x
+    input_shape = x.shape
+    y = jnp.random.randn(*input_shape).astype(jnp.float64)
```

Comment:
There is no `y` in the double function, so there is no need for that either, besides, we do not generate data inside testing functions, we use hypothesis strategies for that, in the decorator